tests/unit/serialization/portable_test.py

```diff
@@ -272,6 +342,25 @@ def create_portable():
         [11, 2, 3],
         [1.0, 2.0, 3.0],
         [11.0, 22.0, 33.0],
+        [decimal.Decimal(12.34), decimal.Decimal(0.5678), decimal.Decimal(0.00000005)],
```

Comment:
can you put some none values here and there in the arrays to test them?

Reply:
It is not possible to put `None` fields in here since array items can not be null. See: [DefaultPortableWriter.java#L331](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/portable/DefaultPortableWriter.java#L331).

I quickly look into other clients and they also don't have `None` tests in here. If this is required, we can create a different class and test case for singular `None` types.